src/diffusers/pipeline_utils.py

```diff
@@ -184,12 +188,18 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike]):
         model_index_dict.pop("_diffusers_version")
         model_index_dict.pop("_module", None)
 
+        def is_saveable_module(name, value):
+            if not isinstance(value, (list, tuple)):
```

Comment:
What if one of the `passed_pipe_kwargs` is a list or a tuple?

Reply:
yeah that's a bit problematic at the moment indeed

Reply:
Changing this!